<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RedactPersonView</class>
 <widget class="QDialog" name="RedactPersonView">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование контакта</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/MainIcons.qrc">
    <normaloff>:/icons/mainIcon/Icons/MainIcons/table.svg</normaloff>:/icons/mainIcon/Icons/MainIcons/table.svg</iconset>
  </property>
  <widget class="QLabel" name="iTelephoneLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Внутренний телефон</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="iTelephoneEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="telegramEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>@</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="snameEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="descrLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дополнительные сведения</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="cTelephoneEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>580</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>580</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telephoneEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="telephoneLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Телефон</string>
   </property>
  </widget>
  <widget class="QLabel" name="cTelephoneLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Городской телефон</string>
   </property>
  </widget>
  <widget class="QLabel" name="telegramLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Телеграм</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pnameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pnameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QLabel" name="snameLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descrEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>251</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>snameEdit</tabstop>
  <tabstop>pnameEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>telegramEdit</tabstop>
  <tabstop>telephoneEdit</tabstop>
  <tabstop>cTelephoneEdit</tabstop>
  <tabstop>iTelephoneEdit</tabstop>
  <tabstop>descrEdit</tabstop>
  <tabstop>acceptButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/MainIcons.qrc"/>
 </resources>
 <connections/>
</ui>
